doc/tools/coqrst/coqdoc/main.py

```diff
@@ -48,28 +48,16 @@ def coqdoc(coq_code, coqdoc_bin=None):
     finally:
         os.remove(filename)
 
-def is_whitespace_string(elem):
-    return isinstance(elem, NavigableString) and elem.strip() == ""
-
-def strip_soup(soup, pred):
-    """Strip elements matching pred from front and tail of soup."""
```

Comment:
Only front, so the docstring was wrong.

Reply:
No I think it also drops elements at the tail (in the first `while` loop).

Reply:
Good point, missed that